scalafmt-tests/src/test/resources/scala3/UnionIntersectionType.stat

```diff
@@ -52,11 +52,9 @@ def hello(): FirstTypeInTheInterSectionType & SecondTypeInTheInterSectionType &
     val a  : FirstTypeInTheInterSectionType & SecondTypeInTheInterSectionType 
 }
 >>>
-def hello(): FirstTypeInTheInterSectionType &
-  SecondTypeInTheInterSectionType &
-  ThirdTypeInTheInterSectionType = {
-  val a: FirstTypeInTheInterSectionType &
-    SecondTypeInTheInterSectionType
+def hello()
+    : FirstTypeInTheInterSectionType & SecondTypeInTheInterSectionType & ThirdTypeInTheInterSectionType = {
```

Comment:
This seems to now exceed 30 column limit, is this correct?

Reply:
correct, but the configuration uses `keep` here.

Reply:
not for infix. infix formatting rules were added about a year ago, they are somewhat complex, and before that the formatter always kept breaks around infix operators as is, regardless of line length.